plugins/reposync.py

```diff
@@ -71,6 +71,8 @@ def set_argparser(parser):
                             help=_('download only newest packages per-repo'))
         parser.add_argument('-p', '--download-path', default='./',
                             help=_('where to store downloaded repositories'))
+        parser.add_argument('--omit-repoid', default=False, action='store_true',
+                            help=_('don\'t add repoid to download-path'))
```

Comment:
The yum implementation of reposync used --norepopath switch for this functionality. Is there any reason not to use it also here?
Also yum allowed --norepopath only when just one repository was about to be synced. I think this restriction makes sense - otherwise repositories content would be mixed together. What do you think?

See https://github.com/rpm-software-management/yum-utils/blob/12dc3da81c8b200fbac15e8fd70d777ff4bb57dc/reposync.py#L108

Reply:
Indeed, you're correct. The original --norepopath does the same. It's better to use that as optionname. Sorry for that, I will change this.